evalml/pipelines/components/estimators/regressors/exponential_smoothing_regressor.py

```diff
@@ -0,0 +1,134 @@
+"""Holt-Winters Exponential Smoothing Forecaster."""
+import numpy as np
+import pandas as pd
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+from evalml.utils import import_or_raise, infer_feature_types
+
+
+class ExponentialSmoothingRegressor(Estimator):
+    """Holt-Winters Exponential Smoothing Forecaster.
+
+    Currently ExponentialSmoothingRegressor isn't supported via conda install. It's recommended that it be installed via PyPI.
+
+    Args:
+        n_jobs (int or None): Non-negative integer describing level of parallelism used for pipelines. Defaults to -1.
+        random_seed (int): Seed for the random number generator. Defaults to 0.
+    """
+
+    name = "Exponential Smoothing Regressor"
+    hyperparameter_ranges = {}
+    """{
+    }"""
+    model_family = ModelFamily.EXPONENTIAL_SMOOTHING
+    """ModelFamily.EXPONENTIAL_SMOOTHING"""
+    supported_problem_types = [ProblemTypes.TIME_SERIES_REGRESSION]
+    """[ProblemTypes.TIME_SERIES_REGRESSION]"""
+
+    def __init__(
+        self,
+        n_jobs=-1,
+        random_seed=0,
+        **kwargs,
+    ):
+        parameters = {}
+        parameters.update(kwargs)
+        smoothing_model_msg = (
+            "sktime is not installed. Please install using `pip install sktime.`"
+        )
+        sktime_smoothing = import_or_raise(
+            "sktime.forecasting.exp_smoothing", error_msg=smoothing_model_msg
+        )
+        smoothing_model = sktime_smoothing.ExponentialSmoothing(**parameters)
+
+        super().__init__(
+            parameters=parameters,
+            component_obj=smoothing_model,
+            random_seed=random_seed,
+        )
+
+    def _remove_datetime(self, data, features=False):
```

Comment:
Nit: Since `_remove_datetime`, `_match_indices`, and `_set_forecast` don't rely on the class instance and are shared by `ARIMARegressor` (and potentially future similar estimators), you can move these to `pipelines/components/utils`. Not necessary for this PR and we can file a separate issue for code cleanup if necessary.